### DISCUSSION TITLE
Settings Storage Type Label

### DISCUSSION THREAD

**Discussion by leepeterson** [pods-framework/pods#6769] [Ideas]
## Issue Overview
Storage Type label "None" for "Custom Settings Page" Type isn't entirely accurate.

## Expected Behavior
Labeling the Storage Type as "Options" seems more accurate.

## Current Behavior
<details>
All Custom Settings Page Storage Type labels basically say there is no storage for options set in a Pods-created Settings page. 
</details>

## Steps to Reproduce (for bugs)
<details>
1. Create New Pod
2. Choose Custom Settings Page
3. Add fields and save
4. View Storage Type column in Manage Pods screen
</details>

## Possible Solution
<details>
* Label the Storage Type as "Options"
* Label the Storage Type as "wp_options"
* Label the Storage Type as "Autoloaded Options"
</details>

## WordPress Environment
<details>
WP 4.9.6 on LEMP stack
</details>

## Pods Package Export (helpful!)
<details>
Pods 2.7.3
</details>

## Workaround or Alternate Solution Until Bug is Addressed
Unknown

## Related Issues and/or PRs
N/A

**Comment by quasel** [comment]
![bildschirmfoto 2018-08-31 um 12 57 17 pm](https://user-images.githubusercontent.com/9464217/44908881-8281c680-ad1d-11e8-9feb-6d2cd7adfe22.png)

Good Catch!
